pom.xml

```diff
@@ -320,6 +320,7 @@
       <plugin>
         <groupId>org.apache.rat</groupId>
         <artifactId>apache-rat-plugin</artifactId>
+        <version>0.13</version>
```

Comment:
Where rat version came from before?

Reply:
Also we have this part in the code:
```
                  <pluginExecutionFilter>
                    <groupId>org.apache.rat</groupId>
                    <artifactId>apache-rat-plugin</artifactId>
                    <versionRange>[0.10,)</versionRange>
```

Reply:
It came from `apache-21.pom` file
```
<plugin>
    <groupId>org.apache.rat</groupId>
    <artifactId>apache-rat-plugin</artifactId>
     <version>0.12</version>
</plugin>
```

That part relates to `lifecycle-mapping` plugin, which is intended for Eclipse m2e settings only. It has no influence on the Maven build itself. However `versionRange` has not upper bound there.